docs/tutorials/models.ipynb

```diff
@@ -51,7 +51,7 @@
    "outputs": [],
```

Comment:
<p>We already have <code>SpatialModel.plot_interactive()</code>,  have tried whether this works. And maybe you could just move the code to <code>SpatialModel.plot_grid()</code>.  I think it's convenient if models can plot themselves...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3083/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='93'/>

Reply:
Done!